protocols/index/contracts/Index.sol

```diff
@@ -119,15 +119,13 @@ contract Index is Ownable {
 
     // Decrement the index length.
     length = length - 1;
```

Comment:
Line 106: ` ) external onlyOwner returns (bool) {`
The function no longer returns a bool.

Line 102: `* @return bool return true on success`
Same